tests/flow/test_index_updates.py

```diff
@@ -27,12 +28,17 @@ def setUpClass(cls):
         cls.r.start()
         redis_con = cls.r.client()
         redis_graph = Graph("index_test", redis_con)
+
+        # cls.r = redis.Redis()
+        # redis_graph = Graph("index_test", cls.r)
+
         cls.populate_graph()
         cls.build_indices()
 
     @classmethod
     def tearDownClass(cls):
         cls.r.stop()
+        # pass
```

Comment:
Why the changes in this file?

Reply:
Adding the ability to run this test against a none disposable redis-server